src/sparc/client/zinchelper.py

```diff
@@ -0,0 +1,116 @@
+from scaffoldmaker.annotation.bladder_terms import get_bladder_term
+from scaffoldmaker.annotation.body_terms import get_body_term
+from scaffoldmaker.annotation.brainstem_terms import get_brainstem_term
+from scaffoldmaker.annotation.colon_terms import get_colon_term
+from scaffoldmaker.annotation.esophagus_terms import get_esophagus_term
+from scaffoldmaker.annotation.heart_terms import get_heart_term
+from scaffoldmaker.annotation.lung_terms import get_lung_term
+from scaffoldmaker.annotation.muscle_terms import get_muscle_term
+from scaffoldmaker.annotation.nerve_terms import get_nerve_term
+from scaffoldmaker.annotation.smallintestine_terms import get_smallintestine_term
+from scaffoldmaker.annotation.stellate_terms import get_stellate_term
+from scaffoldmaker.annotation.stomach_terms import get_stomach_term
+from scaffoldmaker.utils.exportvtk import ExportVtk
+from src.sparc.client.services.pennsieve import PennsieveService
+from mbfxml2ex.app import read_xml
+from mbfxml2ex.zinc import load, write_ex
+
+
+class ZincHelper:
+
+    def __init__(self):
+        self._allOrgan = {
+            "bladder": get_bladder_term,
+            "body": get_body_term,
+            "brainstem": get_brainstem_term,
+            "colon": get_colon_term,
+            "esophagus": get_esophagus_term,
+            "heart": get_heart_term,
+            "lung": get_lung_term,
+            "muscle": get_muscle_term,
+            "nerve": get_nerve_term,
+            "smallintestine": get_smallintestine_term,
+            "stellate": get_stellate_term,
+            "stomach": get_stomach_term,
+        }
+
+        self._context = Context('sparcclient')
+        self._region = self._context.getDefaultRegion()
+        self._pennsieveService = PennsieveService(connect=False)
+
+    def download_files(self, limit=10, offset=0, file_type=None, query=None, organization=None, organization_id=None, dataset_id=None):
+        file_list = self._pennsieveService.list_files(limit, offset, file_type, query, organization, organization_id, dataset_id)
+        try:
+            response = self._pennsieveService.download_file(file_list=file_list)
+        except Exception:
+            raise RuntimeError("The dataset is not downloaded.")
+        assert response.status_code == 200
+        return file_list[0]['name']
+
+    def get_scaffold_vtk(self, dataset_id, output):
+        scaffold_setting_file = self.download_files(limit=1, file_type='JSON', query="Scaffold_Creator-settings.json", dataset_id=dataset_id)
+        with open(scaffold_setting_file) as f:
+            c = json.load(f)
+
+        assert 'scaffold_settings' in c
+        assert 'scaffoldPackage' in c['scaffold_settings']
+
+        sm = scaffolds.Scaffolds_decodeJSON(c['scaffold_settings']['scaffoldPackage'])
+        sm.generate(self._region)
+        ex = ExportVtk(self._region, 'Scaffold VTK export.')
+        ex.writeFile(output)
+
+    def get_mbf_vtk(self, dataset_id, output):
```

Comment:
This API needs to be improved.  
The API should be more like this I think: `dataset_id, dataset_file, output_file=None`.
If the output file is None then the output file should be based on the dataset_file name.